meshtastic/interfaces/ble/interface.py

```
@@ -3183,6 +3164,18 @@ def _send_to_radio_impl(self, toRadio: mesh_pb2.ToRadio) -> None:
 
     def close(self) -> None:
         """Shut down the BLE interface and release associated resources."""
+        # Clear any provisional connecting state before shutdown to prevent
+        # orphaned gate claims when connection threads are forcibly terminated.
+        # This clears all provisional keys owned by this interface, not just
+        # the current self.address value.
+        try:
+            _clear_connecting_for_owner(self)
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)